examples/rest/python/sample_langchain.py

```diff
@@ -2,9 +2,19 @@
 from langchain import LLMChain, PromptTemplate
 from langchain.memory import ConversationBufferWindowMemory
 from langchain.callbacks.streaming_stdout import StreamingStdOutCallbackHandler
-from langchain.document_loaders import TextLoader
+from langchain.document_loaders import TextLoader, UnstructuredRSTLoader, DirectoryLoader
 from langchain.chains.question_answering import load_qa_chain
 from langchain.llms import OpenAI
+from langchain.text_splitter import CharacterTextSplitter
+from langchain.chains import RetrievalQA
+from langchain.vectorstores import Chroma
+
+# Note that Langchain support for embedding documents using MLC is currently blocked on
+# https://github.com/langchain-ai/langchain/pull/7815
+# We have subclassed `OpenAIEmbeddings` in the meantime to get around this dependency.
+from mlc_chat.embeddings.openai import MLCEmbeddings
```

Comment:
@tqchen Subclassed `OpenAIEmbeddings` with a new class `MLCEmbeddings`, lmk if this is what you had in mind.